apps/web/playwright/lib/testUtils.ts

```diff
@@ -170,3 +170,13 @@ export const createNewSeatedEventType = async (page: Page, args: { eventTitle: s
   await page.locator('[data-testid="offer-seats-toggle"]').click();
   await page.locator('[data-testid="update-eventtype"]').click();
 };
+
+export async function assertToasterText(page: Page, locator: string, expectedText: string): Promise<boolean> {
+  try {
+    await expect(page.getByTestId(locator)).toHaveText(expectedText);
+    return true;
+  } catch (error) {
+    console.error(error);
+    return false;
+  }
+}
```

Comment:
As mentioned by @roae there's no need to asset the text as it is prone to change.

```suggestion
```